.changeset/grumpy-carrots-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': major
+---
+
+Accessibility fixes for SelectPanel. This is a **breaking change** for the SelectPanel because there are two new required props (`title` and `inputLabel`).
```

Comment:
@colebemis Should we make these optional until next major?

Reply:
Done, I added reasonable default values so it remains accessible until then.